carbon-design-system/carbon
# docs(file-uploader): show onAddFiles action in storybook

Right now, the [FileUploaderDropContainer story](https://react.carbondesignsystem.com/?path=/story/components-fileuploader--file-uploader-drop-container) is showing `onChange` as the only event in the actions panel.

`onChange` isn't in the prop types though but gives the impression to work because it is emitted on the `<label>` element and therefore is called when the underlying `<input type="file">` fires an `onChange`.

The problem is: this means that `onChange` is only called when the user clicks on the input and selects a file. If the user drags and drops a file on the container, `onChange` is not called.

This PR updates the action in the storybook to show the preferred (and listed in the prop types) `onAddFiles`. which reacts to both: manual selection by clicking and dropping a file.

#### Changelog

**Changed**

- FileUploaderDropContainr story: use `onAddFiles` instead of `onChange` as action

#### Testing / Reviewing

- Visit current react storybook, try to drop a file -> there shouldn't be any action shown in the story
- Run the react storybook with this PR and drop a file -> there should be an action shown: `onAddFiles`
